### PR TITLE
src/trivial-shell.nix: use with to avoid pkgs duplication

### PR DESCRIPTION
**Pull request by SuperSandro2000**
- [ ] Amended the changelog in `release.nix` (see `release.nix` for instructions)

### CODE REVIEW

File: src/trivial-shell.nix [hunk 1]
```diff
@@ -1,10 +1,12 @@
-{ pkgs ? import <nixpkgs> {} }:
+{ pkgs ? import <nixpkgs> { } }:
 
-pkgs.mkShell {
+with pkgs;
```
Comment (grahamc) [inline src/trivial-shell.nix]: I know I'm not very involved with Lorri these days, but my opinion is thaht using `with` in such a large context is an antipattern. I'd be +1 in using for just the list in `buildInputs` though.

File: src/trivial-shell.nix [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+
+mkShell {
   buildInputs = [
```
Comment (nyarly) [inline src/trivial-shell.nix]:
```suggestion
  buildInputs = with pkgs; [
```
?

**Comment by SuperSandro2000** [comment]
done

**Comment by Profpatsch** [comment]
`with` breaks static analysis of nix code.

**Comment by SuperSandro2000** [comment]
What static analysis? I didn't find anything about this in the repo.

**Pull request closed without merging**